src/System.Net.Http/src/MatchingRefApiCompatBaseline.uap.txt

```diff
@@ -0,0 +1,4 @@
+Compat issues with assembly System.Net.Http:
+TypesMustExist : Type 'System.Net.Internal.Cookie' does not exist in the implementation but it does exist in the contract.
```

Comment:
@davidsh can you please confirm whether these should be exposed in the reference assembly?

Reply:
System.Net.Internal.Cookie is a "copy" of System.Net.Cookie and was used in order to access some internal methods.  

I don't understand your question, though.  This is an internal type for System.Net.Http.dll UAP version.  It shouldn't be exposed in any public reference assembly.

Reply:
While they might have Internal in there namespace name they are public types in the implementation, which is what this check is flagging. If there is a reason the need to be public types in the implementation lease let me know and we can keep them baselined otherwise we should consider making them internal in the implementation.

Reply:
Looking at the code, it looks like the classes could be marked 'internal' instead of 'public'.  Right now, the source file is common and there is an ifdef uap which just changes the namespace from System.Net to System.Net.Internal.  We would need to edit the file and add those ifdef uap wrapper to each class definition to change it from public to internal.

Reply:
Thanks @davidsh we could keep this in the baseline file or mark them internal I leave that to you. Could you similarly have a look at the other System.Net baslines in this PR?

Reply:
>Thanks @davidsh we could keep this in the baseline file or mark them internal I leave that to you.

Please just baseline this for now.

cc: @stephentoub @karelz

Reply:
OK I will baseline and file a follow-up issue to track clean-up.